app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CommentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  skip_before_action :authenticate_user!
+  def index
+    post = Post.find(params[:post_id])
+    @comments = post.comments
+    render json: @comments
+  end
+
+  def create
+    user_id = params[:user_id]
+    post_id = params[:post_id]
+    @comment = Comment.new(user_id:, post_id:, text: comment_params[:text])
+
+    if @comment.save
+      render json: 'Comment saved successfully!'
```

Comment:
👍 Good effort on API functionality! 👍 However, there's a minor adjustment needed to meet project requirements:

- [ ] **Valid JSON Response:** While your API endpoints correctly receive JSON, the response currently is not valid JSON. I recommend updating your code to ensure that the API endpoints respond with valid JSON according to project requirements.
![image](https://github.com/Lawmsangi/blog-app/assets/55840999/69617b4e-5afc-4188-b1d7-bfaf6ae46b40)

By ensuring that the response is valid JSON, you'll align with project specifications for API functionality. Keep up the good work! 🚀📊